### PR TITLE
chore: remove addr from datanode error message

### PR DESCRIPTION
**Pull request by paomian**
I hereby agree to the terms of the [GreptimeDB CLA](https://gist.github.com/xtang/6378857777706e568c1949c7578592cc)

## What's changed and what's your intention?
remove addr from datanode error message

## Checklist

- [ ]  I have written the necessary rustdoc comments.
- [ ]  I have added the necessary unit tests and integration tests.

## Refer to a related PR or issue link (optional)
#1132

### CODE REVIEW

**Comment by paomian** [comment]
#1132

**Comment by sunng87** [comment]
As we removed this information from error message, we can seek to log it internally for better debug experience

File: src/client/Cargo.toml [hunk 1]
```diff
@@ -16,6 +16,7 @@ common-grpc-expr = { path = "../common/grpc-expr" }
 common-query = { path = "../common/query" }
 common-recordbatch = { path = "../common/recordbatch" }
 common-time = { path = "../common/time" }
+common-telemetry = { path = "../common/telemetry" }
```
Comment (paomian) [inline src/client/Cargo.toml:19]: I don't know if it is appropriate to introduce logs here.

Comment (MichaelScofield) [reply src/client/Cargo.toml:19]: it's ok

File: src/client/src/error.rs [hunk 2]
```diff
@@ -26,12 +26,7 @@ pub enum Error {
...
 
-    #[snafu(display(
-        "Failed to do Flight get, addr: {}, code: {}, source: {}",
-        addr,
-        tonic_code,
-        source
-    ))]
+    #[snafu(display("Failed to do Flight get, code: {}, source: {}", tonic_code, source))]
     FlightGet {
         addr: String,
```
Comment (MichaelScofield) [inline src/client/src/error.rs:31]: this field can be removed as well

**Review by MichaelScofield — APPROVED** [review]

**Review by sunng87 — APPROVED** [review]
Good job! First blood

File: src/client/src/error.rs (hunk 2 above)
Comment (sunng87) [reply src/client/src/error.rs:31]: We can keep this field for internal logging

Comment (paomian) [reply src/client/src/error.rs:31]: I wanted to delete this field before. Then I thought I might need to use it in the error field as well.

**Pull request merged**